src/components/Camera/CameraView.js

```diff
@@ -30,84 +28,87 @@ type Props = {
 }
 
 const CameraView = ( { camera, device }: Props ): Node => {
-  const zoom = useSharedValue( 0 );
   const [tappedCoordinates, setTappedCoordinates] = useState( null );
-  const tapToFocusAnimation = useRef( new Animated.Value( 0 ) ).current;
+  const singleTapToFocusAnimation = useRef( new Animated.Value( 0 ) ).current;
   // check if camera page is active
   const isFocused = useIsFocused( );
   const isForeground = useIsForeground( );
   const isActive = isFocused && isForeground;
 
-  const minZoom = device?.minZoom ?? 1;
-  const maxZoom = Math.min( device?.maxZoom ?? 1, 5 );
+  // used for pinch and double tap to zoom
+  const scale = useSharedValue( SCALE_MIN_ZOOM );
+  const savedScale = useSharedValue( SCALE_MIN_ZOOM );
+  const animatedStyle = useAnimatedStyle( ( ) => ( {
+    transform: [{ scale: scale.value }]
+  } ) );
 
-  const cameraAnimatedProps = useAnimatedProps( () => {
-    const z = Math.max( Math.min( zoom.value, maxZoom ), minZoom );
-    return {
-      zoom: z
-    };
-  }, [maxZoom, minZoom, zoom] );
-  // #endregion
+  const singleTapToFocus = async ( { x, y } ) => {
+    try {
+      singleTapToFocusAnimation.setValue( 1 );
+      setTappedCoordinates( { x, y } );
+      await camera.current.focus( { x, y } );
+    } catch ( e ) {
+      // Android often catches the following error from the Camera X library
+      // but it doesn't seem to affect functionality
+      // [unknown/unknown] Cancelled by another startFocusAndMetering()] error focusing camera
```

Comment:
How about we throw any error that is *not* that "another startForcusAndMetering" error? Otherwise we're squelching potential exceptions that could tell us about real bugs.